components/navigation/navigation.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="author" content="Sharath Edulakanti">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation | Atom</title>
+    <link rel="stylesheet" href="navigation.css">
+    <link rel="stylesheet" href="../../styles.css">
```

Comment:
Just a suggestion: you can create a `main` css file where you can import every other css stylesheet and add the link of that stylesheet here

Reply:
Thank you for suggestion. Will create a new branch and update  in all components.

Reply:
Yes that's a good suggestion.